engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -109,6 +112,9 @@
  *
  */
 public final class WorldRendererImpl implements WorldRenderer {
+
+    private static final boolean DELAY_INIT = true;
```

Comment:
Yeah `WorldRendererImpl` is still a bit beefy, but it seems like it is getting chipped away at reasonably :-)